packages/template-ui/src/components/DetailView.vue

```diff
@@ -11,16 +11,37 @@
       >
         <span><ArrowLeft />Back</span>
       </b-button>
-      <h1>{{ title }}</h1>
-      <p>{{ subtitle }}</p>
-      <slot>
-      </slot>
+      <b-row>
+        <b-col>
+          <h1>{{ title }}</h1>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col
+          v-if="!bundleKind"
```

Comment:
Please use the `isSimpleBundle` getter instead of using the `bundleKind` state directly.